### ISSUE TITLE
amp to watt

### ISSUE THREAD

**Issue by NOsteyr** [goev/AlsavoProHomeAssistantIntegration#9]
Hi,

The sensor "sensor.alsavopro_NAME_compressor" shows the current (in amps) that the compressor is drawing.

Is it possible to create a new sensor that converts amps to watts at 220V?

I use Home Assistant and know that I can create a sensor for this conversion myself, but I think others might find such a sensor useful too.

I'm assuming a power factor of 1, though it hasn't been accurately measured.

0 amps = 0 watts
1 amp = 220 watts
2 amps = 440 watts
3 amps = 660 watts
4 amps = 880 watts
5 amps = 1,100 watts
6 amps = 1,320 watts
7 amps = 1,540 watts
8 amps = 1,760 watts
9 amps = 1,980 watts
10 amps = 2,200 watts
My pump has never drawn more than 8 amps, but larger units might run much higher.

I assume the power factor is close to 1, but I haven't measured it. The formula is:
Watts = Amps × 220 × Power Factor.

**Comment by gyrex** [comment]
I would advise against this. I've got a hard-wired 32A capable zigbee power meter attached to my pool heater and I can tell you that the compressor current reading has no bearing on the total power usage of the heat pump. For example, when the compressor current is reading 12A @ ~230V, the power usage should be ~2700W but my power meter is showing 3100W. This is probably due to the power factor and other loads outside of the compressor drawing power.

**Comment by goev** [comment]
I agree with gyrex. It's so easy to create this in Home Assistant anyway, so I'll close this.
